widgets/google-map/tpl/static-map.php

```diff
@@ -1,3 +1,9 @@
+<?php
+if( !empty( $destination_url ) ) echo '<a href="' . $destination_url . '" ' . ( $new_window ? 'target="_blank"' : '' ) . '>'; ?>
+
 <img
 	border="0"
-	src="<?php echo sow_esc_url( $src_url ) ?>">
```

Comment:
In general, we want to [escape as late as possible](http://www.wptherightway.org/en/ssl/). Let's remove escaping from `get_template_variables ` and add it to the template files.